### ISSUE TITLE
predicating using Python Code

### ISSUE THREAD

**Issue by moahaimen** [ultralytics/ultralytics#3698]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

@glenn-jocher  I use GOOGLE COLAB when using python code everything works well but  the resulted images of predication doesnt appear or save in any train folder unlike when using CLI its saved in a folder and i can check the bounding boxes confidence
 
also how to make detection for a mp4 video or any youtube video?

# Load a model
model = YOLO("yolov8n.yaml")  # build a new model from scratch
model = YOLO("yolov8n.pt")  # load a pretrained model (recommended for training)

# Use the model
model.train(data="coco128.yaml", epochs=3)  # train the model
metrics = model.val()  # evaluate model performance on the validation set
results = model("https://ultralytics.com/images/bus.jpg")  # predict on an image
path = model.export(format="onnx")  # export the model to ONNX format

**Comment by glenn-jocher** [comment]
@moahaimen hi there!

It's great to hear that you're using YOLOv8 for object detection. In regards to the issue you mentioned, when using the YOLOv8 Python code, by default, the prediction images are not saved to any specific folder. If you'd like to save the resulting images, you can modify the Python code to include the necessary logic to save the images to a desired folder.

Regarding your question about making detections on videos, YOLOv8 supports video detection as well. You can pass the path to an MP4 video or a YouTube video URL to the `model()` function, just like you did for the image prediction. This will allow you to perform object detection on the video and obtain bounding box predictions for each frame.

I hope this helps! Let me know if you have any further questions or need assistance with anything else related to YOLOv8.

Best regards,
Glenn
